packages/did/src/Did.chain.ts

```diff
@@ -60,36 +62,27 @@ export type ChainDidPublicKeyDetails = DidDidDetailsDidPublicKeyDetails
 
 // ### RAW QUERYING (lowest layer)
 
-export function encodeDid(did: DidUri): KiltAddress {
+export function didToChain(did: DidUri): KiltAddress {
   return parseDidUri(did).address
 }
 
-// Query a full DID.
-// Interacts with the DID storage map.
-async function queryDidEncoded(did: DidUri): Promise<Option<DidDidDetails>> {
-  const api = ConfigService.get('api')
-  return api.query.did.did(encodeDid(did))
+export function resourceIdToChain(id: UriFragment): string {
```

Comment:
isn't this a missed opportunity? we could allow DidResourceUri too and simply return the fragment without `#`

Reply:
As discussed, this was not needed before, and would complicate the implementation